src/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -112,6 +113,11 @@ export interface ConversationProps {
 
 const Conversation: React.FC<ConversationProps> = props => {
   const { conversation, relay } = props
+
+  useEffect(() => {
+    UpdateConversation(relay.environment, conversation)
```

Comment:
Since this is a function being called (which matches the mutation field name, versus the type) maybe this should be `updateConversation`?

Reply:
This pattern matches the other mutation that's used in this app. I'd be 👍 for merging this as is and updating them both in a separate PR or updating them both in this PR. @lilyfromseattle whichever you prefer would likely be fine.

Reply:
Sounds good 👍 we should probably update all of them at once so that there's no more copy 🍝 but yeah, better to keep things consistent across app for now.